### DISCUSSION TITLE
Dead Link in Documentation

### DISCUSSION THREAD

**Discussion by hMutzner** [cloud-py-api/nc_py_api#223] [General]
On page https://cloud-py-api.github.io/nc_py_api/reference/Files/Shares.html#nc_py_api.files.sharing.Share.share_with there's is a page missing behind "Refer to the [Sharing examples](https://github.com/cloud-py-api/nc_py_api/tree/main/examples/as_client/sharing)"

Was the page moved to somewhere else. I can't find it.

**Comment by bigcat88** [comment]
Oh, sorry I didn't have time to write them.
I'll try to find time tomorrow and add an examples to the documentation

**Comment by hMutzner** [comment]
Thank you Alexander for your promt answer.

 

Von: Alexander Piskun ***@***.***> 
Gesendet: Samstag, 10. Februar 2024 21:10
An: cloud-py-api/nc_py_api ***@***.***>
Cc: Heinz Mutzner ***@***.***>; Author ***@***.***>
Betreff: Re: [cloud-py-api/nc_py_api] Dead Link in Documentation (Discussion #223)

 

Oh, sorry I didn't have time to write them.
I'll try to find time tomorrow and add an examples to the documentation

**Comment by bigcat88** [comment]
I removed the link and added lines to the description of the classes in which way they can be used, I hope this is enough.

```
Class providing API for creating UI for the ExApp settings, avalaible as **nc.files.sharing.<method>**.
```

APIs itself are strictly parametrised and it should be enough to type only provided path and choose from suggested list by the IDE.
And all params have descriptions that should be visible in IDE.

_Unfortunately, time is very limited; all free time will be spent on expanding the API and finalizing the API for the Nextcloud applications.

This year they can be used as examples.._
